src/api/exporters/exportNQuads.spec.js

```diff
@@ -82,27 +63,30 @@ describe('export Nquads', () => {
         }));
     });
 
-    it.only('should export an object property (with a class)', (done) => {
+    it('should export an object property (with a class)', (done) => {
         let outputString = '';
         exportNQuads(
             {
                 cleanHost: 'https://lodex.data.istex.fr',
                 schemeForDatasetLink: '',
                 '@context': {
-                    ZI8w: { '@id': 'http://data.opendiscoveryspace.eu/lom_ontology_ods.owl#technicalLocation' },
+                    vlM0: { '@id': 'http://purl.org/dc/terms/isPartOf' },
                 },
             },
             fields.slice(1, 2),
             null,
             from([{
-                uri: 'http://data.istex.fr',
-                ZI8w: 'A New Hope' }]),
+                uri: 'http://data.istex.fr/1',
+                D0n9: 'https://publication-type.data.istex.fr/ark:/67375/JMC-0GLKJH51-B' }]),
         ).pipe(ezs((data, feed) => {
             if (data !== null) {
                 outputString += data;
             } else {
                 try {
-                    expect(outputString).toEqual('<http://data.istex.fr> <http://purl.org/dc/terms/title> "A New Hope" .\n');
+                    expect(outputString).toEqual(`<http://data.istex.fr/1/classes/vlM0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://data.istex.fr/ontology/istex#PublicationTypeConcept> .
```

Comment:
This expected output is a proof that the exemple is not yet the best.
A link should not have classes.
The pointed object should declare itself its rdf:type.

Reply:
Morever, the value of the property (`https://publication-type.data.istex.fr/ark:/67375/JMC-0GLKJH51-B`) is not exported in the triples!